docs/getting-started.txt

```diff
@@ -180,130 +188,178 @@ hyperparameters on a single slot.
-default ``slots_per_trial`` setting is ``1``.
+The ``distributed.yaml`` configuration file for this example is the same as the ``const.yaml`` file
+in the previous example, except that a ``resources.slots_per_trial`` field is defined and set to a
+value of ``8``:
 
-For clusters that have multiple GPUs per machine, ``slots_per_trial`` must be divisible by the
-number of GPUs per machine or the job cannot be scheduled. For example, a 7-GPU job cannot be
-scheduled on a 4-GPU per machine cluster, but an 8-GPU job can be scheduled.
+.. code:: yaml
 
-#. Create the experiment using the distributed.yaml file:
+   resources:
+     slots_per_trial: 8
+
+This is the number of available GPU resources. The ``slots_per_trial`` value must be divisible by
+the number of GPUs per machine. You can change the value to match your hardware configuration.
+
+#. Create and run the experiment:
 
    .. code:: bash
 
       det experiment create distributed.yaml .
 
-#. Enter the cluster address in the browser.
+   You can also use the ``-m`` option to specify a remote master IP address:
 
-#. Accept the default determined username with no password and click **Sign In** to view the WebUI
-   dashboard.
+   .. code:: bash
 
-#. Click the **Experiment** tile to view the experiment’s trial display.
+      det -m http://<ipAddress>:8080 experiment create distributed.yaml .
 
-The loss curve is similar to the single-GPU experiment in the previous exercise but the time to
-complete the trial is reduced by about half.
+#. To view the WebUI dashboard, enter the cluster address in your browser address bar, accept the
+   default ``determined`` username, and click **Sign In**. A password is not required.
+
+#. Click the **Experiment** tile to view the experiment’s trial display. The loss curve is similar
+   to the single-GPU experiment in the previous exercise but the time to complete the trial is
+   reduced by about half.
 
 *********************************
  Run a Hyperparameter Tuning Job
 *********************************
 
-Hyperparameter searches involve multiple trials or model variations per experiment. This example
-demonstrates hyperparameter search using the ``adaptive.yaml`` configuration file. The parameter
-settings tell the search algorithm the ranges to explore for each hyperparameter. This example model
-uses a fixed batch size and searches on dropout size, filters, and learning rate. The searcher name
-is set to ``adaptive_asha`` and ``max_trials`` is set to ``16``, indicating which search algorithm
-to use and how many model configurations to explore.
-
-#. Submit the experiment specifying the configuration file:
+This example demonstrates hyperparameter search. The example uses the ``adaptive.yaml``
+configuration file, which is similar to the ``const.yaml`` file in the first example but includes
+additional hyperparameter settings:
+
+.. code:: yaml
+
+   hyperparameters:
```

Comment:
I recommend using the `includeliteral` thing that I used in the core API docs to keep one source of truth for this code block.

This recommendation applies throughout the doc, there are other codeblocks which are copy/pasted but which don't have to be.

Reply:
I think that might work for reference docs but I'm less inclined to do that for non-reference docs. The manual and mental effort needs to be put into maintaining this kind of content without having a (somewhat) hidden dependency. Using ``literalinclude`` doesn't hide the maintenance issue, particularly if the whole file isn't included.